src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/DisposeAnalysis/DisposeAnalysisHelper.cs

```diff
@@ -145,6 +149,16 @@ public bool HasAnyDisposableCreationDescendant(ImmutableArray<IOperation> operat
                 HasDisposableOwnershipTransferForConstructorParameter(containingMethod);
         }
 
+        public bool IsDisposableTypeNotRequiringToBeDisposed(ITypeSymbol typeSymbol) =>
```

Comment:
Name is a little verbose but explain well (IMO) the goal. Let me know if you prefer something else.